src/components/Card.js

```diff
@@ -17,42 +33,92 @@ class Card {
     this._cardImageEl = this._cardElement.querySelector(".card__image-card");
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._title;
+    this._cardLikesEl = this._cardElement.querySelector(".card__button-count");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**